src/systems/painter.js

```diff
@@ -77,10 +77,12 @@ AFRAME.registerSystem('painter', {
     if (urlParams.url || urlParams.urljson) {
       var isBinary = urlParams.urljson === undefined;
       this.brushSystem.loadFromUrl(urlParams.url || urlParams.urljson, isBinary);
-      document.getElementById('logo').setAttribute('visible', false);
-      document.getElementById('acamera').setAttribute('orbit-controls', 'position', '0 1.6 3');
-      document.getElementById('apainter-logo').classList.remove('hidden');
-      //document.getElementById('apainter-author').classList.remove('hidden'); // not used yet
+      document.addEventListener('DOMContentLoaded', e => {
```

Comment:
This is strange. This logic is in the init method. It shouldn't run before the DOM is ready. Maybe I'm not fully understanding